phpcs.xml

```diff
@@ -1,8 +1,36 @@
 <?xml version="1.0"?>
 <ruleset name="Laminas coding standard">
-    <rule ref="./vendor/laminas/laminas-coding-standard/ruleset.xml"/>
-
     <!-- Paths to check -->
     <file>src</file>
     <file>test</file>
+
+    <rule ref="LaminasCodingStandard">
+        <exclude name="WebimpressCodingStandard.Files.DeclareStrictTypes.BelowComment"/>
```

Comment:
This rule must be removed from the coding standard.

Reply:
Should we wait for an upstream release, or is this to be merged first?

Reply:
Yes, that's why I just released a new coding standard version.

Reply:
Done.